### PR TITLE
Add return type to getMarkPrice

### PR DESCRIPTION
**Pull request by lobobruno**
Mark price return type

https://binance-docs.github.io/apidocs/futures/en/#mark-price

### CODE REVIEW

**Comment by lobobruno** [comment]
Sorry for the amount of PRs lately. I was about the re-write my Binance-Conector (from JS to TS) and I found yours. I'm starting to rewrite my app using your connector and when I come across  something that I'm using that it's not here yet, I'm trying to do my best and contribute !!

File: src/usdm-client.ts [hunk 1]
```diff
@@ -153,7 +154,9 @@ export class USDMClient extends BaseRestClient {
     return this.get('fapi/v1/markPriceKlines', params);
   }
 
-  getMarkPrice(params?: Partial<BasicSymbolParam>): Promise<any> {
+  getMarkPrice(
+    params?: Partial<BasicSymbolParam>
+  ): Promise<MarkPrice | MarkPrice[]> {
```
Comment (tiagosiebler) [inline src/usdm-client.ts:159]: By the way, if you wanted to get a smart return type, I recently discovered this is one way of getting the correct return type (depending on whether the symbol param is included:
https://github.com/tiagosiebler/bybit-api/blob/master/src/spot-client.ts#L89-L95

Doesn't have to be in this PR, but it's a nice touch

**Comment by tiagosiebler** [comment]
> Sorry for the amount of PRs lately. I was about the re-write my Binance-Conector (from JS to TS) and I found yours. I'm starting to rewrite my app using your connector and when I come across something that I'm using that it's not here yet, I'm trying to do my best and contribute !!

No need to apologise! This is great! These are basically types I'd need sooner or later but haven't had the chance to add yet. I also love the fact the PRs are small and bite-size, it's a super easy review. Made a small suggestion but happy to merge as it.

If you do make more PRs in future (which I absolutely welcome), would you mind also bumping the package.json version? That'll make my life easier for the release process, assuming I approve and merge the PR. Thanks!

**Review by tiagosiebler — APPROVED** [review]

**Comment by lobobruno** [comment]
> If you do make more PRs in future (which I absolutely welcome), would you mind also bumping the package.json version? That'll make my life easier for the release process, assuming I approve and merge the PR. Thanks!

Absolutely!

**Pull request merged**